images/aleph/modules/aleph-dev.nix

```diff
@@ -46,6 +46,8 @@ in {
       pkgs.nvidia-jetpack.l4t-camera
     ];
   };
+  environment.etc."elodin-version".text = "0.13.3";
```

Comment:
Can this be read from the root `Cargo.toml`?
e.g. `(craneLib.crateNameFromCargoToml {cargoToml = ../../../Cargo.toml;}).version`